fronted/paintball-app/.gitignore

```diff
@@ -2,4 +2,4 @@
 .env
 
 # Ignore node_modules directory
-node_modules/
\ No newline at end of file
+node_modules/
```

Comment:
Did you make sure that the old `node_modules` is now not part of the repo anymore (check it in GitHub for example)?